node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -532,22 +533,21 @@ public void updateContainerNodeMetrics() {
         // CPU usage by a container as percentage of total host CPU, cpuPercentageOfHost, is given by dividing used
         // CPU time by the container with CPU time used by the entire system.
         // CPU usage by a container as percentage of total CPU allocated to it is given by dividing the
-        // cpuPercentageOfHost with the ratio of container resources over total host resources. This calculation
-        // assumes that the ratio between container and host resources for disk, memory, and cpu is roughly equal
-        // and therefore only calculates the ratio of container memory against host memory.
+        // cpuPercentageOfHost with the ratio of container minCpuCores by total number of CPU cores.
         double cpuPercentageOfHost = lastCpuMetric.getCpuUsagePercentage(cpuContainerTotalTime, cpuSystemTotalTime);
-        double cpuPercentageOfAllocated = getInverseContainerShareOfHost(nodeSpec) * cpuPercentageOfHost;
+        Optional<Double> cpuPercentageOfAllocated = nodeSpec.minCpuCores.map(containerNumCpuCores ->
```

Comment:
Yes, the "complexity" is e.g. interpreting a CPU core we promise as part of a flavor. E.g. knowing that if you have 48 cores according to the flavor, and see ~100% CPU utilization, but you actually will get about 1 core less. E.g. perhaps we shouldn't promise 48 CPU flavor for C-77E, etc, etc.